src/routes/forgot-password/components/ForgotPassword.js

```diff
@@ -70,30 +72,41 @@ module.exports = class ForgotPassword extends StrangeForms(React.Component) {
 
         return (
 
-            <form onSubmit={this.submit}>
-                <h2>Forgot your password?</h2>
-                <p>To reset your password, enter your email below and we will email a link to reset your password.</p>
-                <div>
-                    <label>Email:</label>
-                    <input
-                        id='email'
-                        value={this.fieldValue('email')}
-                        onChange={this.proposeNew('email')}
-                        onBlur={this.fieldBlurred}
-                    />
-                    {this.showEmailError() &&
-                        <label style={{ color:'red' }}>Please enter a valid email address</label>
-                    }
-                </div>
-                {this.props.errorMessage &&
-                    <div style={{ color: 'red' }}>Error! {this.props.errorMessage}</div>
-                }
-                <button
-                    type='submit'
-                    onClick={this.submit}
-                    disabled={this.disableButton()}
-                >Reset Password</button>
-            </form>
+            <FormWrapper>
+                <form onSubmit={this.submit}>
+                    <TextWrapper>
+                        <Typography variant='headline' gutterBottom>Forgot your password?</Typography>
+                        <Typography>Enter your email below and we will email you a link to reset your password.</Typography>
+                        <TextField
+                            id='email'
+                            variant='outlined'
+                            label='Email'
+                            margin='normal'
+                            fullWidth
+                            value={this.fieldValue('email')}
+                            onChange={this.proposeNew('email')}
+                            onBlur={this.fieldBlurred}
+                            error={this.showEmailError()}
+                        />
+                        {this.showEmailError() &&
+                            <FormHelperText>
+                                Please enter a valid email address.
+                            </FormHelperText>
```

Comment:
Should I give `<FormHelperText>` something like `color='error'` to make the error message red, or should I leave it the default grey color?

Reply:
Good question -- more conventions thoughts -- would be a good topic for a group of us to think about,

We could wrap the `FormHelperText` in a styled-component and pass an `error` property, which sets the color to red, or just set the `color='error'` on all error fields

Reply:
In the [docs](https://material-ui.com/api/form-helper-text/#formhelpertext) it looks like there's already a `bool` prop called `error` that turns these red; I can just do that!

Reply:
Cool

Reply:
I think having the convention that error messages are red seems sensible

Reply:
I'll make it so 👍